addons/l10n_cl/models/account_move.py

```diff
@@ -131,7 +131,13 @@ def _get_last_sequence_domain(self, relaxed=False):
 
     def _get_name_invoice_report(self):
         self.ensure_one()
-        if self.l10n_latam_use_documents and self.company_id.account_fiscal_country_id.code == 'CL':
+        if (
+            self.l10n_latam_use_documents and self.company_id.account_fiscal_country_id.code == "CL"
+            and (
+                self.move_type in {"out_invoice", "out_refund"}
+                or (self.move_type == "in_invoice" and self.l10n_latam_document_type_id.code == "46")
```

Comment:
```suggestion
                or self.l10n_latam_document_type_id.code == "46"
```
This might be simpler?